ee/codegen_integration/fixtures/faa_q_and_a_bot/display_data/faa_q_and_a_bot.json

```diff
@@ -2029,7 +2029,46 @@
             "nodes",
             "final_output_2"
           ]
-        }
+        },
+        "outputs": [
+          {
+            "id": "8c6e5464-8916-4039-b911-cf707855d372",
+            "name": "node_input",
```

Comment:
[blocking] I would expect all of these to match the outputs class attribute, so `"name": "value"`